static/news/news.md

```diff
@@ -1,6 +1,18 @@
 ####Please read the [Wiki/FAQ](https://github.com/pymedusa/SickRage/wiki) before opening an issue####
 All issues and bug reports must be opened at [GitHub](https://github.com/pymedusa/SickRage/issues)
 
+####2017-02-19####
+
+## 0.1.11
+
+Little time has passed, but a lot of work has been done. With the most noticeable changes regarding user interface improvements. We'd like to thank wimpyrbx for his awesome work on the new logo, as well as the many designs he has provided which have resulted in a new look & feel. Helping shaping up Medusa to what it's now. You can expect to see even more UI improvements in the coming months.
```

Comment:
"We'd like to thank wimpyrbx for his awesome work on the new logo, as well as the many designs he has provided which have resulted in a new look & feel. Helping shaping up Medusa to what it's now."

Add a capital for "wimpyrbx".

Remove the "Helping shaping up Medusa to what it's now." as it just drags on the last sentence.